internal/exec/stages/files/units.go

```diff
@@ -17,37 +17,92 @@ package files
 import (
 	"fmt"
 	"path/filepath"
+	"strconv"
 	"strings"
 
+	"github.com/coreos/go-systemd/dbus"
```

Comment:
This is an external package, we usually split those imports to a different newline-separated stanza (see other source files for examples).